### ISSUE TITLE
[5.x] Files not uploaded with "Use AJAX" disabled

### ISSUE THREAD

**Issue by Romanavr** [solspace/craft3-freeform#1126]
**Describe the bug or issue you're experiencing**
Upgraded forms (from the 4.x) with the file uploads fields not shown anymore in the submission.
Basically, if you'll upload a file & submit in the form which created on the 4.x uploaded file not shown, however, it's actually uploaded are sent to in the email template.
I checked and the created new form -> the file is shown in submission.
I also tried duplicating the upgraded form and changing the file upload location, file still not shown in submission.

**Steps to reproduce**
1. Try uploading the file in one of the upgraded forms
2. The file is not visible in submission

**Expected behavior**
1. The file should be shown in submission

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.8.0
- Freeform Version: 5.1.2
- Freeform Edition: **Pro**
- Fresh Install or Upgrade: **Upgrade**

**Comment by Romanavr** [comment]
**UPD**:  I found what the problem is: with the option turned off, the files are not uploaded.
I don't think that's how it's supposed to work?
![изображение](https://github.com/solspace/craft-freeform/assets/3519951/e8c3d46c-581a-4f77-9187-99ad112371b4)

**Comment by kjmartens** [comment]
Hi @Romanavr,

Sorry for the trouble you're experiencing.

I tested this and your theory, and I can't seem to duplicate the issue. Are you seeing any JS errors in the browser console? And if so, can you share them, as well as the Network Response error logged? 🙂

**Comment by Romanavr** [comment]
Hi @kjmartens 

Watch this video please:
https://drive.google.com/file/d/1KjoYmGA08nk0Nk7OTDQCt6PaHJSYzyB3/view?usp=sharing
This is a new craft site that only has a CMS and Freeform in the PRO version. For comparison, I added a "drag & drop" field
**I was able to reproduce. What am I doing wrong?**
I didn’t show it on the video, but there were no error logs in free form.

Template used to render form:
```
<html>
    <head>
        <title>Testing forms</title>
    </head>
    <body>
        {{ freeform.form("testForm").render() }}
    </body>
</html>
```

**Comment by kjmartens** [comment]
Ok, I see the issue now @Romanavr.

Thank you for the clarification. I've logged the issue to be fixed shortly. 🙂

**Comment by kjmartens** [comment]
This is now resolved in Freeform 5.1.5. 🙂

**Comment by johnvanhulsen** [comment]
@kjmartens I'm running the latest version (5.1.14) on Craft 4.8.9 but we've got the same issues as above.

```twig
{{ contentField.form.render({
	attributes: {
		class: 'freeform-form',
	},
	buttons: {
		attributes: {
			submit: { class: "btn btn--secondary" },
			back: { class: "btn btn-secondary" },
		},
	},
}) }}
```

Output:
```html
<div class="freeform-row">
<div data-field-container="bestand" data-field-type="file-dnd" class="freeform-column-12 freeform-fieldtype-file-dnd freeform-column"><label class="freeform-label freeform-required" for="form-input-bestand">Bestand</label>
<div novalidate="" class="freeform-input freeform-file-dnd__input" data-freeform-file-upload="bestand" data-error-append-target="bestand" data-file-count="0" data-max-files="1" data-max-size="2048000" data-theme="light" data-message-progress="Bezig met uploaden..." data-message-complete="Upload compleet!" data-message-files="Maximale uploadlimiet van 1 bereikt" data-message-size="Maximale bestandsuploadgrootte is 2048KB" data-accent="#3a85ee" data-base-url="https://localdomain.tld/freeform/" style="--accent: #3a85ee;"><div data-placeholder="" class="freeform-file-dnd__placeholder">Drag and drop files here or click to upload</div><div data-preview-zone="" class="freeform-file-dnd__preview-zone"></div><ul data-messages="" class="freeform-file-dnd__messages"></ul><input type="file" id="form-input-bestand" multiple=""></div></div>
</div>
```

After submit:
<img width="264" alt="image" src="https://github.com/solspace/craft-freeform/assets/2969578/05277a6b-4c35-4653-8e2e-8f454f32c650">
